test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,18 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        // null object
+        assertTrue(Utils.isAnyNull((Object) null));
```

Comment:
Have you taken into account all possible cases? (Given the method name you have used) If you are still not clear on this, post on the slack forum.